lib/pbench/test/functional/server/test_datasets.py

```diff
@@ -554,8 +585,15 @@ def test_contents(self, server_client: PbenchServerClient, login_user):
 
         for dataset in datasets:
             response = server_client.get(
-                API.DATASETS_CONTENTS, {"dataset": dataset.resource_id, "target": ""}
+                API.DATASETS_CONTENTS,
+                {"dataset": dataset.resource_id, "target": ""},
+                raise_error=False,
             )
+            archive = dataset.metadata["server.archiveonly"]
+            if archive:
+                assert response.status_code == HTTPStatus.CONFLICT
+                return
```

Comment:
Um...shouldn't that be a `continue` at line 595 and not a `return`?!

We don't really want to abort the test on the first `archiveonly` result, do we??

Reply:
Um, yeah. Where the heck were you last week, anyway? 😦

Reply:
Sorry, Dave!  🥴